### PR TITLE
Updated icon_dev_guide-ko.md

### PR DESCRIPTION
**Pull request by Life4honor**
Fixed typos, incorrect url and Seperated "Installing Tbears with Docker" into 2 ways (1. MacOS, 2. Ubuntu Linux)

### CODE REVIEW

File: docs/icon_dev_guide-ko.md [hunk 1]
```diff
@@ -216,7 +218,7 @@ source venv/bin/activate
 ```
 * 생성된 `dist` 디렉토리의 `.whl` 파일과 `pip` 를 이용하여 T-Bears를 설치합니다.
 ```
-(venv) pip install ./dist/tbears-1.0.5-py3-none-any.whl
+(venv) pip install ./dist/tbears-1.0.5.1-py3-none-any.whl
```
Comment (Spl3en) [inline docs/icon_dev_guide-ko.md]: As the version is supposed to be updated frequently, what about using this generic command instead : 

``` (venv) pip install ./dist/tbears-*-py3-none-any.whl ```

Comment (Life4honor) [reply docs/icon_dev_guide-ko.md]: Thanks for giving me a feedback about versioning. 👍

File: docs/icon_dev_guide-ko.md [hunk 2]
```diff
@@ -218,7 +218,7 @@ source venv/bin/activate
 ```
 * 생성된 `dist` 디렉토리의 `.whl` 파일과 `pip` 를 이용하여 T-Bears를 설치합니다.
 ```
-(venv) pip install ./dist/tbears-1.0.5.1-py3-none-any.whl
+(venv) pip install ./dist/tbears-*.1-py3-none-any.whl
```
Comment (Spl3en) [inline docs/icon_dev_guide-ko.md]: `*.1` will only match versions that finishes with ".1", which won't work in future releases.
A better version would be : 
```(venv) pip install ./dist/tbears-*-py3-none-any.whl```

Comment (Life4honor) [reply docs/icon_dev_guide-ko.md]: 👍

**Pull request merged**